easygui/boxes/button_box.py

```diff
@@ -396,6 +396,7 @@ def configure_root(self, title):
         self.boxRoot.protocol('WM_DELETE_WINDOW', self.x_pressed)
         self.boxRoot.bind("<Escape>", self.cancel_pressed)
         self.boxRoot.iconname('Dialog')
+        self.boxRoot.attributes("-topmost", True)  # Put the dialog box in focus.
```

Comment:
I'd say it is always the desired behavior. It's the behavior for some of the dialogs already with no apparent reason as to which does and which doesn't (I think it's whenever you have a preselected button, the behavior puts it on top and in focus). EasyGUI is about dialog boxes rather than a general GUI window toolkit, so the "display and put it in focus" should be the intended behavior. I think the dialog boxes that didn't do that were an oversight.